py_jama_rest_client/client.py

```diff
@@ -629,6 +630,11 @@ def post_item(self, project, item_type_id, child_item_type_id, location, fields)
             "fields": fields
         }
         resource_path = 'items/'
+        # we setting a global ID?
+        if global_id is not None:
+            body['globalId'] = global_id
+            resource_path += '?setGlobalIdManually=true'
```

Comment:
Instead of manually setting the param value here.  Use the Requests library param method that handles formatting of request parameters.

params = {}
if resource_path is not None:
  params['setGlobalIdManually] = true